website/.gitignore

```diff
@@ -1,5 +1,6 @@
 # Dependencies
 /node_modules
+package-lock.json
```

Comment:
is this related?

Reply:
Nah, but this is really annoying to me (I have to use npm instead of yarn), don't think creating another PR for just that one line is worth it